go/libraries/doltcore/sqle/tables.go

```diff
@@ -32,6 +35,11 @@ import (
 	"github.com/liquidata-inc/dolt/go/store/types"
 )
 
+const (
+	partitionMultiplier = 2.0
+	minRowsPerPartition = 1024
```

Comment:
Probably want to make this configurable for testing. Otherwise all the engine / unit tests will always have a single partition.